XML/TestCases/Other.xml

```diff
@@ -51,4 +51,14 @@
         <Area></Area>
         <Tags>gcov</Tags>
     </test>
+    <test>
+        <testName>LONG-PERF-STRESS-TEST</testName>
```

Comment:
why not StressTest.xml?

Reply:
this is long term testing and not to be executed as part of regular stress tests, so it's fine in the other.xml